### PR TITLE
[bitnami/postgresql-ha]Adds SSL connections between pgpool and postgr…

### PR DESCRIPTION
**Pull request by rafariossaa**
…esql

**Description of the change**

This PR adds support to SSL in pgpool<->posgresql communications

**Benefits**

This will make the deployment more secure in some environments.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/postgresql-ha/Chart.yaml [hunk 1]
```diff
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 7.5.4
+version: 7.6.0
```
Comment (juan131) [inline bitnami/postgresql-ha/Chart.yaml:30]: Please fix the "no new line at end of the file" warning. You should have your IDE configured to add it automatically.

File: bitnami/postgresql-ha/README.md [hunk 2]
```diff
@@ -160,6 +160,12 @@ The following table lists the configurable parameters of the PostgreSQL HA chart
...
+| `postgresql.tls.certFilename`                   | Certificate filename                                                                                                                                                                            | `""`                                                         |
+| `postgresql.tls.certKeyFilename`                | Certificate key filename                                                                                                                                                                        | `""`                                                         |
+| `postgresql.tls.certCAFilename`                 | CA Certificate filename. If provided, PgPool will authenticate TLS/SSL clients by requesting them a certificate.                                                                                | `nil`                                                        |
```
Comment (juan131) [inline bitnami/postgresql-ha/README.md:168]: Is Pgpool conscious of all this? It looks to me like you're only using these certificates in the PostgreSQL side

**Review by juan131 — CHANGES_REQUESTED** [review]
The README.md should include a section explaining how to enable TLS for this solution. Preferably, it should point to a section at https://docs.bitnami.com where it's properly documented.

File: bitnami/postgresql-ha/README.md (hunk 2 above)
Comment (rafariossaa) [reply bitnami/postgresql-ha/README.md:168]: Pgpool will connect to the backend using TLS once those parameters are enabled.
For example, this is the output when only enabling TLS in postgresql:

```
$ kubectl run mypg-postgresql-ha-client --rm --tty -i --restart='Never' --namespace default --image docker.io/bitnami/postgresql-repmgr:11.12.0-debian-10-r14 --env="PGPASSWORD=$POSTGRES_PASSWORD"  \
>         --command -- psql -h mypg-postgresql-ha-pgpool -p 5432 -U postgres -d postgres
If you don't see a command prompt, try pressing enter.

postgres=# 
postgres=# 
postgres=# pgpool show ssl;
 ssl 
-----
 off
(1 row)

postgres=# select * from pg_stat_ssl;
 pid | ssl | version |         cipher         | bits | compression | clientdn 
-----+-----+---------+------------------------+------+-------------+----------
 273 | f   |         |                        |      |             | 
 276 | f   |         |                        |      |             | 
 281 | t   | TLSv1.3 | TLS_AES_256_GCM_SHA384 |  256 | f           | 
 363 | t   | TLSv1.3 | TLS_AES_256_GCM_SHA384 |  256 | f           | 
 362 | t   | TLSv1.3 | TLS_AES_256_GCM_SHA384 |  256 | f           | 
 438 | f   |         |                        |      |             | 
 271 | f   |         |                        |      |             | 
 270 | f   |         |                        |      |             | 
 272 | f   |         |                        |      |             | 
(9 rows)
```

**Comment by rafariossaa** [comment]
> The README.md should include a section explaining how to enable TLS for this solution. Preferably, it should point to a section at https://docs.bitnami.com where it's properly documented.

Created PR https://github.com/bitnami/charts-docs/pull/40

File: bitnami/postgresql-ha/README.md (hunk 2 above)
Comment (rafariossaa) [reply bitnami/postgresql-ha/README.md:168]: Pgpool will connect to the backend using TLS once those parameters are enabled.
For example, this is the output when only enabling TLS in postgresql:

```
$ kubectl run mypg-postgresql-ha-client --rm --tty -i --restart='Never' --namespace default --image docker.io/bitnami/postgresql-repmgr:11.12.0-debian-10-r14 --env="PGPASSWORD=$POSTGRES_PASSWORD"  \
>         --command -- psql -h mypg-postgresql-ha-pgpool -p 5432 -U postgres -d postgres
If you don't see a command prompt, try pressing enter.

postgres=# 
postgres=# 
postgres=# pgpool show ssl;
 ssl 
-----
 off
(1 row)

postgres=# select * from pg_stat_ssl;
 pid | ssl | version |         cipher         | bits | compression | clientdn 
-----+-----+---------+------------------------+------+-------------+----------
 273 | f   |         |                        |      |             | 
 276 | f   |         |                        |      |             | 
 281 | t   | TLSv1.3 | TLS_AES_256_GCM_SHA384 |  256 | f           | 
 363 | t   | TLSv1.3 | TLS_AES_256_GCM_SHA384 |  256 | f           | 
 362 | t   | TLSv1.3 | TLS_AES_256_GCM_SHA384 |  256 | f           | 
 438 | f   |         |                        |      |             | 
 271 | f   |         |                        |      |             | 
 270 | f   |         |                        |      |             | 
 272 | f   |         |                        |      |             | 
(9 rows)
```

File: bitnami/postgresql-ha/Chart.yaml [hunk 3]
```diff
@@ -27,4 +27,5 @@ name: postgresql-ha
...
+
```
Comment (juan131) [inline bitnami/postgresql-ha/Chart.yaml:31]: "Nor Mr. John nor little Johny", now you included an extra unneeded empty line

File: bitnami/postgresql-ha/README.md (hunk 2 above)
Comment (juan131) [reply bitnami/postgresql-ha/README.md:168]: All right, thanks!

**Review by juan131 — APPROVED** [review]
LGTM! Please wait for the changes in docs.bitnami.com to be deployed before merging the PR

**Pull request merged**